src/init.cpp

```diff
@@ -605,15 +604,17 @@ bool AppInit2(ThreadHandlerPtr threads)
 
     fDebug=false;
 
-    if (fDebug)
-        fDebugNet = true;
-    else
-        fDebugNet = GetBoolArg("-debugnet");
-
```

Comment:
This code never worked anyway! It is before the pull from GetArg... ! But I implemented the intent in the change...

Reply:
`LogFlags::NET` is pretty chatty now since it inherited some entries from `debug10`. May want to drop this in the future.

Reply:
Maybe we should just drop it now, since someone can always turn the category on if they want. I think I will do that.

Reply:
Took it out.